static_global/js/main.js

```diff
@@ -63,6 +63,11 @@ let error = '';
 			if(!result || !result[1]) return message('Reader could not understand the given link.', 1);
 			result = '/read/catbox/' + result[1];
 			break;
+        case /rawkuma\.com/.test(text):
+            result = /rawkuma\.com\/(manga\/)?[A-Za-z0-9-]+/i.exec(text);
+            if(!result) return message('Reader could not understand the given link.', 1);
+            result = '/rk/' + text;
+            break;

```

Comment:
nit: can you change the indentation to use tabs here? I think the original file used tabs so it's probably best if we keep it consistent.